docs/geedocs/5.2.4/answer/7160004.html

```diff
@@ -69,6 +70,23 @@ <h5>
             <th class="narrow">Description</th>
             <th>Resolution</th>
           </tr>
+          <tr>
+            <td>881</td>
+            <td>Error Saving Map Layer with Icon</td>
+            <td>
+              The registry of image decoders in the third-party skia library,
+              used in handling icons for map layers, is now being populated.
+              Saving map layers with icons now works.
```

Comment:
can we reword "now works" to "now functions correctly"?

Reply:
ok